src/Maestro/SubscriptionActorService/PullRequestDescriptionBuilder.cs

```diff
@@ -0,0 +1,201 @@
+using Maestro.Data.Models;
+using Microsoft.DotNet.DarcLib;
+using Microsoft.Extensions.Logging;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+using static SubscriptionActorService.PullRequestActorImplementation;
+
+namespace SubscriptionActorService
+{
+    public class PullRequestDescriptionBuilder
+    {
+        public PullRequestDescriptionBuilder(ILoggerFactory loggerFactory, string description)
```

Comment:
[nit] we should add documentation for this constructor, especially to make it clear what the description string is supposed to be.